tests/test_dumpscript.py

```diff
@@ -78,22 +79,21 @@ def test_with_datetimefield(self):
             club=django,
         )
 
-        dumpscript_path = './django_extensions/scripts'
+        dumpscript_path = Path(__file__).parent.parent / 'django_extensions' / 'scripts'
+        dumpscript_path.mkdir(parents=True, exist_ok=True)
+        (dumpscript_path / '__init__.py').touch()
 
-        os.mkdir(dumpscript_path)
-        open(dumpscript_path + '/__init__.py', 'w').close()  # for python 2.7
+        # Delete dumpscript at the end of the test
+        self.addCleanup(shutil.rmtree, dumpscript_path)
```

Comment:
I learned about `enterContext` from [this Adam Johnson's post](https://adamj.eu/tech/2024/12/30/python-temporary-files-directories-unittest/) and seemed like it would help here.

As the folder needs to be in a specific location, the solution from the post wasn't exactly what I needed, but that helped me find this `addCleanup` method which is used under the hood by `enterContext`.